README.md

```diff
@@ -29,23 +29,52 @@ For more information about Medic Mobile's architecture and how the pieces fit to
 For more information about the format of docs in the database, see [Database Schema](https://github.com/medic/medic-docs/blob/master/development/db-schema.md).
 For more information about the SMS exchange protocol between webapp and gateway, see [Message States](https://github.com/medic/medic-docs/blob/master/user/message-states.md).
 
+## Using the Core Framework
+
+If you wish to evaluate the core framework, _or_ you are a developer looking to create or modify core framework configurations, you should use the [easy deployment](#easy-deployment) instructions.
+
+If you are a developer looking to contribute to the core framework itself, you should follow the [development setup instructions](#development-instructions).
+
+If you're looking to modify the default configuration or upload your own you will need to also familiarise yourself with [medic-conf](https://github.com/medic/medic-conf). A brief guide for modifying the config is available [alongside the config](./config/default/GUIDE.md). A more detailed guide is available in [medic-docs](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md).
+
+### Supported Browsers
+
+Currently, the latest versions of Chrome, Chrome for Android and Firefox are functionally supported. We do not support Safari (unreliable implementations of necessary web APIs) and the generic android browser (unreliable implementations in general). Our webapp code, which includes any code written as configuration, is still ES5. Our exact support matrix (including older app versions) can be found [in our docs](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md).
+
 ## Easy Deployment
 
-To get up and running quickly, [you can use Docker](https://github.com/medic/medic-docs/blob/master/installation/public-docker-image-setup.md). You can then [create your own custom application](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md), or set up the standard application by running [the Medic Configurer](https://github.com/medic/medic-conf) on the [./config/standard](https://github.com/medic/medic/tree/master/config/standard) directory.
+We are going to use Docker to start a Core Framework environment on your local machine.
```

Comment:
That was 100% my intention, just in this section :-)

I wanted to be as hand-holdey as possible, because it's the section for our technically least adept readers. This is why it goes to *much* more detail than we'd normally ever do, and it breaks my personal rule about only explaining things that are new to your situation (ie for developers we don't explain how to install CouchDB as they have instructions for that, but in this section I explain _how to open a terminal_).

With an understanding that this was the intention, is that OK?

Reply:
Ah, thanks for clarifying. Changing the tone back and forth in the same doc is confusing so I'd recommend pulling this out into a separate file. We can still make it clear in the _Using the Core Framework_ intro section to divide the audience based on what they are intending to do with the Core Framework.